docs/docs/administration/backup-and-restore.md

```diff
@@ -64,15 +88,54 @@ gunzip < db_dumps/last/immich-latest.sql.gz | docker exec -i immich_postgres psq
 Immich stores two types of content in the filesystem: (1) original, unmodified content, and (2) generated content. Only the original content needs to be backed-up, which includes the following folders:
 
 1. `UPLOAD_LOCATION/library`
-1. `UPLOAD_LOCATION/upload`
-1. `UPLOAD_LOCATION/profile`
+2. `UPLOAD_LOCATION/upload`
+3. `UPLOAD_LOCATION/profile`
+
+### Asset Types and Storage Locations
+
+<Tabs>
+  <TabItem value="storage template engine OFF (Default)." label="storage template engine OFF (Default)." default>
+
+:::note
+`\library\library` folder is not used by default on new machines running version 1.92.0. These are if the system administrator activated the storage template engine, for [more info](https://github.com/immich-app/immich/releases#:~:text=the%20partner%E2%80%99s%20assets.-,Hardening%20storage%20template,-We%20have%20further).
```

Comment:
We normally use `/` for path, assuming the majority of users are using Linux environment